addons/guide_10_app_exclusions/manifest.json

```
@@ -0,0 +1,38 @@
+{
+    "api_version": "0.1",
+    "id": "guide_10_app_exclusions",
+    "name": "Guide: app exclusions",
+    "type": "guide",
+    "conditions": {
+        "min_client_version": "2.14.0",
+        "enabled_features": ["splitTunnel"]
+    },
+    "guide": {
+        "id": "app_exclusions",
+        "title": "Why you should use app exclusions",
+        "advanced": true,
+        "image": "qrc:/addons/guide_10_app_exclusions/image.svg",
+        "blocks": [
+            {
+                "id": "c_2",
+                "type": "text",
+                "content": "By default, when you activate Mozilla VPN on your device, all the apps you have in it will have their traffic be automatically encrypted and routed by the VPN."
```

```suggestion
                "content": "By default, when you activate Mozilla VPN on your device, all the apps you have in it will have their traffic automatically encrypted and routed by the VPN."
```

@chrisrooneys 
That "be" feels unnecessary?